.github/workflows/container-image.yml

```diff
@@ -117,6 +184,9 @@ jobs:
   test-nightly-ad-server-kubernetes:
       #runs-on: ubuntu-latest
       # need to explicitly use 20.04 to avoid problems with jq...
+      needs:
+        - build-nightly-ad-server
+        - build-nightly-server
```

Comment:
Putting the lines here splits the comment above from the line it's remarking upon (the runs-on line below).
This probably needs to be shifted up or down to avoid this splitting.

Reply:
Fixed.